src/kibali/ProtectedResource.cs

```diff
@@ -406,7 +428,7 @@ private static void ProcessPermissionPairs(HashSet<(string, string)> permissionP
             }
             if (higherPrivilegedPairs.Count != 0)
             {
-                higher = string.Join(", ", higherPrivilegedPairs.OrderByDescending(x => x.Length));
+                higher = string.Join(", ", higherPrivilegedPairs.OrderBy(x => x));
```

Comment:
@wandeg, Is the ordering here necessary? We still have a huge diff with this change. https://github.com/microsoftgraph/microsoft-graph-docs/pull/24946/files

Reply:
Updated